src/components/v5/common/ActionSidebar/hooks/useGetActionData.ts

```diff
@@ -17,6 +17,12 @@ import {
 import { DecisionMethod } from './useDecisionMethods';
 import { useGetColonyAction } from './useGetColonyAction';
 
+const mapExternalLinks = (data: ExternalLink[] | undefined | null) =>
+  data?.map(({ name, link }) => ({
+    linkType: name,
+    url: link,
+  }));
+
```

Comment:
This mapping is unnecessary and it causes the blank table after creating an action. If you need to omit the typename, see `excludeTypenameKey` util.